views/templates.pug

```diff
@@ -478,6 +478,7 @@
   #tree-editor-tpl
     li.list-group-item.tree-editor-file.input-group-append
         span.filename someresource.txt
-        button.btn.delete-file.fa.fa-trash
-        button.btn.rename-file.fa.fa-tag
-        button.btn.stage-file.fa.fa-plus
+        button.btn.delete-file.fa.fa-trash(title="Remove this file" aria-label="Delete")
```

Comment:
Comments welcome on the text here! I started playing with the UI to do a "howto" doc and thought 'ooh these should have tooltips!'